### DISCUSSION TITLE
[4] User doesn't know what to do after installing a new language

### DISCUSSION THREAD

**Discussion by simbus82** [joomla/joomla-cms#47130] [2. Cold Features]
### Steps to reproduce the issue

Install a language.

### Expected result

User needs to be informed about how to enable the languages. For example give a link to the /index.php?option=com_languages&view=installed and add some advices in the success message.

### Actual result

User is informed only about the successful installation of language pack.
Information is missing in the help modal too.
![image](https://user-images.githubusercontent.com/1375730/130269447-e1176f07-121b-4155-86c3-98edb0cd745e.png)

**Comment by PhilETaylor** [comment]
Installing "Language packs" is done by the generic installer and this success message is also a concatenation and sprintf of several strings. The installer would be the wrong place to add a language specific message directing the user to the docs.

**Comment by simbus82** [comment]
> Installing "Language packs" is done by the generic installer and this success message is also a concatenation and sprintf of several strings. The installer would be the wrong place to add a language specific message directing the user to the docs.

Clear but i never writed about redirecting user to docs...
In the "language installation process" done by the user, this the "moment" when we need to give a good information to what to do and how to do next steps. If we can't do it in the page, we can add more infos in the "help page", that now only describe the fields in the page.

**Comment by richard67** [comment]
Closing as having a pull request. Please test #42610 . Thanks in advance.

**Comment by alikon** [comment]
the pr #42610 it's ok
but doesn't solve this, "issue", imho better place on https://github.com/joomla/joomla-cms/discussions

**Comment by brianteeman** [comment]
It is very difficult to propose a solution here as there are at least three different use cases for installing a language
1. You want to be able to manage Joomla in that language
2. You want to create content in that language
3. You want to create content in multiple languages.
